content/en/docs/concepts/secure-deployment.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Securely deploying Notation"
+description: "Best practices for securely deploying Notation"
+type: docs
+weight: 2
+---
+
+A deployment of Notation, specifically the `notation` CLI, is used for signing and verifying container images. The `notation` CLI is usually deployed on a development computer or using a CI/CD pipeline, such as GitHub Actions.
+
+A deployment has several sensitive components:
+
+- Security policies
+- Certificates
+- Trust stores
+- Plugins
+
+It is strongly recommended that you follow the best practices in this document to secure your deployment of Notation.
+
+## Secure filesystem access
+
+The sensitive components of a deployment of Notation are stored in on the filesystem. For development computers and hosts directly managed by your organization, you should follow your organizations best practices for securing those development computers and hosts, such as:
```

Comment:
nit:
`in on the filesystem` ==> `in the filesystem`

Reply:
nice catch, fixed